legacy/build-deploy-docker-compose.sh

```diff
@@ -377,11 +377,8 @@ declare -A IMAGES_PROMOTE
 # this array stores the hashes of the built images
 declare -A IMAGE_HASHES
 
-HELM_ARGUMENTS=()
+# this sets CAPABILITIES which is used by some processes in this build 
 . /kubectl-build-deploy/scripts/kubectl-get-cluster-capabilities.sh
-for CAPABILITIES in "${CAPABILITIES[@]}"; do
-  HELM_ARGUMENTS+=(-a "${CAPABILITIES}")
-done
```

Comment:
Unrelated changes?

Reply:
yeah unrelated. these were used when we were still using helm templates